build/schemas/urdf-model.json

```diff
@@ -25,6 +25,12 @@
                                     "title": "Url",
                                     "type": "string"
                                 },
+                                "urlBase": {
+                                  "description": "Base path for xacro/urdf package includes. This must be an absolute path with no trailing slash, e.g. '/store/users/username/robot'",
```

Comment:
Do we need to handle dropbox here? Come to think of it, do we handle nested files from dropbox in .gltf, or only .glb for dropbox?

Reply:
I guess `crossOriginDropboxSrc` util function would be needed